web/src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -267,7 +270,7 @@ export default defineComponent({
   },
   setup(props, ctx) {
     const location = ref(rootDirectory);
-    const itemToDelete = ref(null);
+    const itemToDelete = ref();
```

Comment:
I don't see a need to type this as `any`

```suggestion
    const itemToDelete = ref<AssetFile | null>(null);
```

Reply:
You're right, that's the correct type; I also had to make some additional changes to satisfy the type checker due to the limited support for typescript in Vue templates.

On another note unrelated to this PR, just curious, I noticed you used this syntax for declaring the ref type
`const itemToDelete = ref<AssetFile | null>(null);`
instead of
`const itemToDelete: Ref<AssetFile | null> = ref(null);`.

Is there any specific reasoning behind that? I've been using the latter because feels cleaner to me and is similar to the syntax for Python type hints + other languages I've worked with that are actually statically typed, but I've seen both syntaxes used and I'm not sure what the advantages of using one over the other are.